README.md

```diff
@@ -63,12 +60,12 @@ Instructions for building the model in multiple steps are provided below.
 ## How to Set Up (Configure) and Run GEOSldas
 
 
-a) At **NCCS**, GEOSldas must be built, configured, and run on the same operating system. To run GEOSldas on Milan nodes (SLES15), start with `ssh discover-mil`.
+a) To run GEOSldas on Milan nodes (SLES15), start with `ssh discover-mil`.
```

Comment:
Do we need the `-mil` flag?  Presumably, when Discover comes back on Thursday, `ssh discover` should be fine